build-system/compile/build.conf.js

```diff
@@ -90,15 +90,16 @@ function getReplacePlugin() {
     }
   });
 
-  // default each backup experiment constant to false as well
+  // default each backup experiment constant to the customized value
   Object.keys(experimentsConstantBackup).forEach(experimentDefine => {
     function flagExists(element) {
       return element['identifierName'] === experimentDefine;
     }
 
     // only add default replacement if it already doesn't exist in array
     if (experimentDefine && !replacements.some(flagExists)) {
-      replacements.push(createReplacement(experimentDefine, false));
+      const value = !!experimentsConstantBackup[experimentDefine];
```

Comment:
nit: use `Object.entries()` to iterate instead

Reply:
ha didn't know that. Good point : )